docs/basics/connection-approval.md

```diff
@@ -25,6 +25,7 @@ This is how the connection approval response is formed by server-side specific u
 - **PlayerPrefabHash**: The type of player prefab to use for the authorized player (_if this is null it uses the default `NetworkManager` defined player prefab_)
 - **Position** and **Rotation**: The position and rotation of the player when spawned
 - **Pending**: Provides the ability to mark the approval "pending" to delay the authorization until other user-specific code finishes the approval process.
+- **Reason**: If `Approved` is `false` you can populate this with a `string` based message (or `json`) to send the reason why the client was not approved.
 
 :::note
 Unlike previous versions of Netcode for GameObjects where users were provided a callback to be invoked within the connection approval handler method, users now only need to set the appropriate properties of the `NetworkManager.ConnectionApprovalResponse` class.  Part of this update allows users to set their `ConnectionApprovalResponse` to `Pending` which provides users additional time to process any other tasks involved with the player approval process.
```

Comment:
Unlike previous versions of Netcode for GameObjects, where users had to invoke a callback within the connection approval handler method, users now only need to set the appropriate properties of the `NetworkManager.ConnectionApprovalResponse` class. Part of this update allows users to set their `ConnectionApprovalResponse` to `Pending`, which provides users additional time to process any other tasks involved with the player approval process.